e3sm_to_cmip/default.py

```diff
@@ -41,6 +42,8 @@ def write_data(varid, data, timeval=None, timebnds=None, index=None, **kwargs):
             else:
                 outdata = data[RAW_VARIABLES[0]].values
 
+        outdata[np.isnan(outdata)] = 1e20 
```

Comment:
```suggestion
        outdata[np.isnan(outdata)] = 1e-20 
```

Good catch @TonyB9000, it should be 1e-20 if we're dealing with small floating points here. @chengzhuzhang you can commit this suggestion directly if this sounds right.

Reply:
Please reconsider. This is not a value we get to choose. The CMIP6 standard `_FillValue` is `1.e20`, not `1.e-20`, and for good reason.

Reply:
Right Charlie - Thanks.  I don't know why I saw "-20" somewhere.  I assume "1e20" is an acceptable way to represent 1.e20 (or 1.e+20) in xarry-speak.

Reply:
Thank you for looking over!
1e20 was used in other instances when assigning missing values in e3sm_to_cmip. I believe 1e20 is equivalent to  1.e20 (or 1.e+20).

The meta data for variable ra is now:
```
	float ra(time, lat, lon) ;
		ra:standard_name = "surface_upward_mass_flux_of_carbon_dioxide_expressed_as_carbon_due_to_plant_respiration" ;
		ra:long_name = "Carbon Mass Flux into Atmosphere Due to Autotrophic (Plant) Respiration on Land [kgC m-2 s-1]" ;
		ra:comment = "Carbon mass flux per unit area into atmosphere due to autotrophic respiration on land (respiration by producers) [see rh for heterotrophic production]" ;
		ra:units = "kg m-2 s-1" ;
		ra:cell_methods = "area: mean where land time: mean" ;
		ra:cell_measures = "area: areacella" ;
		ra:missing_value = 1.e+20f ;
		ra:_FillValue = 1.e+20f ;
```
And missing values are properly represented with `-`